### ISSUE TITLE
Accessibility contrast issues

### ISSUE THREAD

**Issue by rickybassom** [autoreduction/queue-processor#790]
Issue raised by: developer

### What?
In many places in the web app there's low contrast between text and background colors which should be improved for accessibility. The "Take a tour" button for example.

### Where?
Web app

### How?
Investigating this issue https://github.com/ISISScientificComputing/autoreduce/issues/786 and running the Wave tool.

### Reproducible?
yes

### How to test the issue is resolved
* Install the [Wave tool](https://wave.webaim.org/) browser extension.
* Run wave tool against the web page which will tell you the areas of low contrast and fix them.

**Comment by rickybassom** [comment]
A lot of the contrast issues are inherited from Bootstrap. It might be better to upgrade Bootstrap before looking at this. https://github.com/ISISScientificComputing/autoreduce/issues/644

**Comment by rickybassom** [comment]
As a temporary fix for the buttons on the overview page we could change the background colour to #347AB7 as suggested by @dtasev
